src/decanter/core/jobs/data_upload.py

```diff
@@ -25,6 +25,7 @@ class DataUpload(Job):
     """DataUpload manage to get the result from data upload.
 
     Handle the execution of upload task in order to upload data to
+
```

Comment:
what is it

Reply:
not sure what you mean. Should I remove the white line?

Reply:
yes :)